pkg/util/util.go

```diff
@@ -338,5 +338,9 @@ func IsSmallerQuantity(requestStr, limitStr string) (bool, error) {
 		return false, fmt.Errorf("could not parse limit %v : %v", limitStr, err2)
 	}
 
+	if limitStr == "0" { // Zero disables setting a limit
+		return false, nil
+	}
```

Comment:
not sure about this. Maybe we should better lift the `required` setting in the CRD schema in another PR.

Reply:
To the best of my memory, the reason for this conditional was to prevent the code after from executing because `return request.Cmp(limit) == -1, nil` would always, given a non-zero request and a zero (null) limit, return false even though setting a request but not a limit is perfectly valid, legitimate, and potentially desirable behavior.

As for the required setting I assume you're referring to [this](https://github.com/zalando/postgres-operator/blob/32743e2058f0e7e7895cb1a36f7f873d965d82bd/manifests/postgresql.crd.yaml#L411)? I'm not sure how that's related though.